cluster/ci/config/prow/openshift/jenkins_operator.yaml

```diff
@@ -250,6 +260,10 @@ items:
     ports:
     - port: 80
```

Comment:
If you name one port, you need to name them all. I think I used `main` for these